.devcontainer/Dockerfile

```diff
@@ -0,0 +1,9 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.188.0/containers/cpp/.devcontainer/base.Dockerfile
+
+ARG VARIANT="buster"
+FROM mcr.microsoft.com/vscode/devcontainers/cpp:0-${VARIANT}
+
+RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
```

Comment:
I imagine this global variable works on both Debian and Ubuntu?

Reply:
yes.

Reply:
Disables any _input_ required for commands if i recall correctly